images/macos/templates/macOS-10.15.json

```diff
@@ -196,6 +196,13 @@
             "execute_command": "chmod +x {{ .Path }}; {{ .Vars }} pwsh -f {{ .Path }}",
             "scripts": "./provision/core/toolset.ps1"
         },
+        {
+            "type": "shell",
+            "execute_command": "chmod +x {{ .Path }}; {{ .Vars }} {{ .Path }}",
+            "scripts": [
+                "./provision/core/pipx-packages.sh"
```

Comment:
Please move this script to the section where `pypy.sh` is located. It doesn't make sense to create the new one

Reply:
Moved it